__tests__/topics.test.js

```diff
@@ -0,0 +1,36 @@
+const app = require("../api/app");
+const request = require("supertest");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  it("GET: 200 sends an array to the client", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topics)).toBe(true);
+      });
+  });
+
+  it("GET: 200 return array is populated with objects that have the required properties - 'slug' and 'description'", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
```

Comment:
Are you sure you're getting an array of all the topics back from your response? We know it's an array and that each item has those properties, but is it the right length?